assets/stylesheets/layout/_authentication.scss

```diff
@@ -0,0 +1,22 @@
+.layout-authentication {
```

Comment:
Minor comment on the directory name as it should be `layouts/` to be consistent with `components/`

Reply:
Fixed in 12f96b87968c3853494f8e01a055260ec940b3c0